packages/celotool/src/lib/bls_utils.ts

```diff
@@ -0,0 +1,84 @@
+const keccak256 = require('keccak256')
+const ecurve = require('ecurve')
+const BigInteger = require('bigi')
+const reverse = require('buffer-reverse')
+
+const Curve = ecurve.Curve
+const Point = ecurve.Point
+
+const p = BigInteger.fromHex(
```

Comment:
Are these parameters for a specific, named curve that we chose? Might be useful to add a comment with such a name for future reference.

Reply:
Good point. I'll do that.